apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -0,0 +1,294 @@
+module.exports = {
+  backend: {
+    name: "git-gateway",
+    branch: "main",
+  },
+  media_folder: "public/images",
+  public_folder: "/images",
+  local_backend: true,
+  collections: [
+    {
+      name: "pages",
+      label: "Pages",
+      files: [
+        {
+          label: "Index",
+          name: "index",
+          widget: "object",
+          file: "content/pages/index.md",
+          fields: [
+            {
+              label: "Hero",
+              name: "hero",
+              widget: "object",
+              collapsed: true,
+              fields: [
+                {
+                  label: "Title",
+                  name: "title",
+                  widget: "markdown",
+                },
+                {
+                  label: "Messages",
+                  name: "messages",
+                  widget: "list",
+                },
+                {
+                  label: "Subtitle",
+                  name: "subtitle",
+                  widget: "text",
+                },
+                {
+                  label: "Image",
+                  name: "image",
+                  widget: "image",
+                },
+              ],
+            },
+            {
+              label: "Partners",
+              name: "partners",
+              widget: "relation",
+              collection: "partners",
+              search_fields: ["name"],
+              value_field: "logo.src",
+              display_fields: ["name"],
+              multiple: true,
+            },
+          ],
+        },
+      ],
+    },
+    {
+      name: "partners",
+      label: "Data | Partners",
```

Comment:
```suggestion
      label: "Data | Partners",
      label_singular: "Partner"
```

So that the button show "New Partner" instead of "New Data | Partners"